sql/core/src/test/resources/sql-tests/results/postgreSQL/window_part2.sql.out

```diff
@@ -72,17 +59,6 @@ struct<sum(unique1) OVER (PARTITION BY four ORDER BY unique1 ASC NULLS FIRST RAN
 8	6	2
 
 
--- !query
-select last(salary) over(order by salary range between 1000 preceding and 1000 following),
-lag(salary) over(order by salary range between 1000 preceding and 1000 following),
-salary from empsalary
--- !query schema
-struct<>
--- !query output
-org.apache.spark.sql.AnalysisException
-Window Frame specifiedwindowframe(RangeFrame, -1000, 1000) must match the required frame specifiedwindowframe(RowFrame, -1, -1);
```

Comment:
We also have many test cases to check exception error messages to prevent any unintentional behavior changes (both good or bed).

Reply:
This query works fine in PostgreSQL. Sorry I missed some contexts in PostgreSQL test porting but shouldn't we match the results and queries?

The purpose of PostgreSQL in this query is positive but I wonder if it's right to change it to negative test case for this one alone.

Reply:
@dongjoon-hyun @HyukjinKwon According your discussion, I will revert this test case back.